### ISSUE TITLE
VQGAN training details

### ISSUE THREAD

**Issue by Andrew-Brown1** [CompVis/taming-transformers#61]
Hi,

Thanks for the great repo! Could I ask some questions about training VQGAN? 

What batch size did you train it with, and for how long? 
Also I see here that you wait until you add the discriminator loss https://www.youtube.com/watch?v=fy153-yXSQk

Do you wait until the model has converged without it before adding it? 

Thanks!

**Comment by rromb** [comment]
Hi, great question :) Most of our published VQGAN models are trained on a single 40GB VRAM GPU with a batch size of ~12 (bs=14 for the f16 model), depending on the hyperparameters. Regarding your second question, yes, it makes sense to monitor the perceptual loss and then add the discriminator to the training loop.

**Comment by Andrew-Brown1** [comment]
Hey - thanks!

**Comment by gombru** [comment]
Hi! Can we get any details about training costs? I.e. how many epochs and gpu days took to train the OpenImages model on a single 40GB VRAM GPU?

Thanks!
